cypress-io/cypress
# fix: Ensure tooltips are displayed behind modals

- Closes #22595 

### User facing changelog
Ensure tooltips for overlaid content do not appear in front of modal dialogs

### Additional details

### Steps to test
Little bit tricky, but the easiest way to verify this is to get a tooltip to stick open - typically you can throw your mouse around the ACI column headers (Latest Runs & Average Duration) and eventually one will stay open after you've moused off (it'll close if you mouse back over or onto another item with a tooltip).

If you have trouble getting a tooltip to stick open you can try adding `:shown="true"` to `Tooltip.vue` around line 12 which should force at least one tooltip open without any interaction.

Once you get a tooltip stuck open, click the "New Spec" button and verify that the tooltip does not overlay the modal or its backdrop.

### How has the user experience changed?
**Before**

![Screen Shot 2022-07-27 at 11 09 53 AM](https://user-images.githubusercontent.com/11482842/181323895-ad61dd23-7019-4dee-a3c5-d971f2c67213.png)

**After**

![Screen Shot 2022-07-27 at 11 09 33 AM](https://user-images.githubusercontent.com/11482842/181324191-45a9f277-8b27-4d1f-97bf-956db55daff2.png)

### PR Tasks
